pages/blog/google-season-of-docs-2022.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "📑 Google Season of Docs 2022 at AsyncAPI"
+date: 2022-03-25T06:00:00+01:00
+type: Engineering
+tags:
+  - Documentation
+  - Announcement
+cover: /img/posts/gsod-2022/SeasonofDocs_Logo.webp
+authors:
+  - name: Alejandra Quetzalli
+    photo: /img/avatars/canela-ale.webp
+    link: https://www.linkedin.com/in/alejandra-quetzalli/
+    byline: Our proposal? Update Docs Information Architecture
+excerpt: Check out the Docs project proposal we're submitting to GSoD 2022! You won't want to miss out.
+featured: true
+---
+
```

Comment:
Sadly, I have removed the emojis that were in _headings_ because they make our heading links angry 😡 🤣 . That's why the right-hand nav TOC links weren't working before.